lib/bucketed_game.nit

```diff
@@ -32,6 +35,8 @@ end
 # Something acting on the game from time to time
 class Bucketable[G: Game]
 	super Turnable[G]
+	auto_serializable
```

Comment:
You need to repeat it even if the superclass is already tagged as `auto_serializable`?